test/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(IOScancoTests
 
 CreateTestDriver(IOScanco  "${IOScanco-Test_LIBRARIES}" "${IOScancoTests}")
 
+if(0) # HACK: Missing download files for testing
```

Comment:
@thewtex @dzenanz This commented out code allows building, but comments out tests due to missing test data content links.

Reply:
I uploaded this file -- will remove this commit from the branch and push.